index.html

```diff
@@ -947,620 +974,1301 @@
+                <div class="container pr-0 pt-1">
+                  <div class="row pb-2">
+                    <div class="col-lg-4 px-0 pt-1">
+                      <img
+                        src="./assets/trump-kwame.JPG"
+                        alt="trump lowers sentence for kwame"
+                        class="img-fluid Us-small-img"
+                      />
+                    </div>
+                    <div class="col-lg-8 Us-small-text pr-0 pt-1">
+                      <strong
+                        >'Tiger King' Zoo Owners Ordered To Hand Over Big Cats
+                        to Government</strong
+                      >
+                    </div>
+                  </div>
                 </div>
               </div>
             </div>
-            <div class="nstorty">
-              <img src=./assets/cindy-mccain-doug-ducey-funeral.jpg alt="biden"
-              class="images-float-left">
-              <div class="heading_text">
-                <div class="bheading">
-                  Warren Calls for Unity 'Against Insurrection' as Trump
-                  Impeachment Delayed
+          </div>
+        </div>
+        <div class="row">
+          <div class="col-lg-12 px-0 subscribe-container border-top pt-4">
+            <div class="container px-0">
+              <div class="row">
+                <div class="col-lg-9 pr-4">
+                  <p class="subscribe-title">SUBSCRIBE</p>
+                  <div class="container background-gainsboro pr-2 mr-4">
+                    <div class="row">
+                      <div class="col-lg-5 py-5 pl-4">
+                        <img
+                          class="img-fluid"
+                          src="./assets/subscribe-newsletter-img.JPG"
+                          alt="newsletter subscription image"
+                        />
+                      </div>
+                      <div class="col-lg-7 py-5 mt-2">
+                        <strong class="subscribe-title-description">
+                          In-depth Stories You Don't See Elsewhere About Topics
+                          You Want to Read!
+                        </strong>
+                        <p class="subscribe-description mt-3">
+                          All Access + Weekly Delivery. Choose A Membership
+                          That's Perfect for You!
+                        </p>
+                        <button
+                          class="subscribe-button button-tag-magazine-page px-2 rounded"
+                        >
+                          SUBSCRIBE
+                        </button>
+                      </div>
+                    </div>
+                  </div>
                 </div>
-                <div class="text">
-                  "Unity starts with accountability," Warren said Sunday during
-                  an interview with CNN's Abby Phillip.
+                <div class="col-lg-3 pl-0">
+                  <p class="subscribe-title">NEWSLETTER</p>

```

Comment:
- [ ] This should be a header tag(h2, h3, h34, etc), that guarantees the code is more semantic and accessible to screen readers.